### PR TITLE
Run tests against PHP 7.3

### LINKED ISSUE #217: Broken on PHP 7.3: "continue" targeting switch is equivalent to "break"
"continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"? {"userId":"NqntwzG58vk","exception":"[object] (ErrorException(code: 0): \"continue\" targeting switch is equivalent to \"break\". Did you mean to use \"continue 2\"? at vendor/smalot/pdfparser/src/Smalot/PdfParser/PDFObject.php:329)

### PR DESCRIPTION
**Pull request by fbourigault**
~~ATM, failure is allowed.~~

Fix #217

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,11 +8,13 @@ php:
   - 7.0
   - 7.1
   - 7.2
+  - 7.3
   - hhvm
   
 matrix:
   allow_failures:
     - php: hhvm
+    - php: 7.3
```
Comment (j0k3r) [inline .travis.yml]: I think this shouldn't be allowed

**Comment by j0k3r** [comment]
@smalot Can you take a look? :pray:

File: .travis.yml (hunk 1 above)
Comment (fbourigault) [reply .travis.yml]: I allowed it to fail as 7.3 isn't released yet.

**Comment by belgattitude** [comment]
Possible to merge soon ? Thanks a lot.

**Comment by j0k3r** [comment]
@smalot 👋

**Comment by j0k3r** [comment]
Thanks for merging it @smalot
Any chance to publish it on packagist (and maybe enable auto-update)?
https://packagist.org/packages/smalot/pdfparser

Thanks again.

**Comment by smalot** [comment]
Should be fixed.
Thanks for notifying it

**Comment by belgattitude** [comment]
Thanks @smalot !

**Pull request merged**